blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+A recently introduced Python package enables distributing ROOT RDataFrame workloads to a set of remote resources.
+This feature is available in experimental phase since the latest ROOT 6.24 release, allowing users to write and run
+their applications from within the same interface while steering the computations to an Apache Spark cluster.
+
+## In detail
```

Comment:
For this title, what about: `One programming model, many backends`

Reply:
Not sure. It's not the point of the section. I agree that "In detail" is generic, but I think it has to be because we're not describing only the programming model in this text. Still, I do not prefer the current title either so I am willing to change it if others also agree